test/components/patch_serviceaccount_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aks aws aws_edge packet
+// +build e2e
+
+// nolint:testpackage
```

Comment:
Why not `components_test` for package name?

Reply:
There is already a package named components. Go throws error

Reply:
it seems to work for me:
```sh
0 ✓ (11.4s) 23:38:28 invidian@dellxps15mateusz ~/repos/kinvolk/lokomotive (pr/704)*$ git diff
diff --git a/test/components/patch_serviceaccount_test.go b/test/components/patch_serviceaccount_test.go
index 9c309ffd..f1c7b987 100644
--- a/test/components/patch_serviceaccount_test.go
+++ b/test/components/patch_serviceaccount_test.go
@@ -15,8 +15,7 @@
 // +build aks aws aws_edge packet
 // +build e2e

-// nolint:testpackage
-package components
+package components_test

 import (
        "context"
0 ✓ (44.9ms) 23:38:33 invidian@dellxps15mateusz ~/repos/kinvolk/lokomotive (pr/704)*$ make build-test
go test -run=nonexistent -mod=vendor -tags="aws,packet,aks,e2e,baremetal,disruptivee2e,poste2e" -covermode=atomic -buildmode=exe -v ./... > /dev/null
0 ✓ (3.987s) 23:38:38 invidian@dellxps15mateusz ~/repos/kinvolk/lokomotive (pr/704)*$
```